app/scripts/views/reset_password.js

```diff
@@ -8,6 +8,7 @@ define(function (require, exports, module) {
   const AuthErrors = require('lib/auth-errors');
   const BaseView = require('views/base');
   const Cocktail = require('cocktail');
+  const FlowBeginMixin = require('views/mixins/flow-begin-mixin');
```

Comment:
Another scenario I can think of is if the user opens their email on another device, currently no `flowId` is set, with these changes. Previously, a new `flowId` was set. Is this a scenario where we might need to stash/gather metrics?